hack/deploy/prepare-int-parameters.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash -e
+mkdir -p secrets
+cat >secrets/parameters.json <<EOF
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "databaseAccountName": {
+            "value": "$COSMOSDB_ACCOUNT"
+        },
+        "domainName": {
+            "value": "$DOMAIN_NAME"
+        },
+        "extraCosmosDBIPs": {
+            "value": ""
+        },
+        "extraKeyvaultAccessPolicies": {
+            "value": []
+        },
+        "fpServicePrincipalId": {
+            "value": "$AZURE_FP_CLIENT_ID"
+        },
+        "keyvaultPrefix": {
+            "value": "$KEYVAULT_PREFIX"
+        },
+        "pullSecret": {
+            "value": "$PULL_SECRET"
+        },
+        "rpImage": {
+            "value": "$RP_IMAGE"
+        },
+        "rpImageAuth": {
+            "value": "$RP_IMAGE_AUTH"
+        },
+        "mdmFrontendUrl": {
+            "value": "$MDM_FRONTEND"
+        },
+        "mdsdConfigVersion": {
+            "value": "$MDSD_CONFIG_VERSION"
+        },
+        "mdsdEnvironment": {
+            "value": "$MDSD_ENVIROMENT"
+        },
+        "rpMode": {
+            "value": "$RP_MODE"
+        },
+        "rpServicePrincipalId": {
+            "value": "SET-LATER"
+        },
```

Comment:
better to remove this rather than inserting a dummy value?  it will highlight an error in use earlier that way.  Ditto vmssName below.